src/plugins/plugin-http.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+var shimmer = require('shimmer');
+var url = require('url');
+var isString = require('is').string;
+var merge = require('lodash.merge');
+var httpAgent = require('_http_agent');
+
+function patchModuleRoot (request, api) {
+  var labels = api.labels;
```

Comment:
nit: given that there is another variable below named `labels` that shadows this, it would be better not to have this variable to begin with and use `api.labels` everywhere.